client/src/app/mpegts_info_table.rs

```diff
@@ -81,5 +122,93 @@ impl MpegTsInformationsTable {
+            stream.stream_info.packets.iter().for_each(|packet| {
+                packet.content.fragments.iter().for_each(|fragment| {
+                    let header = &fragment.header;
+                    // TODO: handle multiple streams
+                    if let PIDTable::ProgramAssociation = header.pid {
+                        if aggregator.is_pat_complete() {
+                            let info = MpegTsInfo::PatBuffer(&aggregator.pat_buffer);
+                            let key = RowKey {pid: header.pid, alias: stream.alias.clone()};
+                            if mpegts_rows.contains_key(&key) {
+                                let val = mpegts_rows.get_mut(&key).unwrap();
+                                val.counter += 1;
+                                val.time = packet.time;
+                            } else {
+                                mpegts_rows.insert(
+                                    key,
+                                    MpegTsInfoRow {
+                                        info,
+                                        counter: 0,
+                                        source_addr: packet.packet_association_table.source_addr,
+                                        destination_addr: packet.packet_association_table.destination_addr,
+                                        time: packet.time
+                                    });
+                            }
+                        }
+                    } else if let PIDTable::PID(pid) = header.pid {
+                        if aggregator.is_pmt_complete(pid) {
+                            let info = MpegTsInfo::PmtBuffer(
+                                aggregator.pmt_buffers.get(&pid).unwrap()
+                            );
+                            let key = RowKey {pid: header.pid, alias: stream.alias.clone()};
+                            if mpegts_rows.contains_key(&key) {
+                                let val = mpegts_rows.get_mut(&key).unwrap();
+                                val.counter += 1;
+                                val.time = packet.time;
+                            } else {
+                                mpegts_rows.insert(
+                                    key,
+                                    MpegTsInfoRow {
+                                        info,
+                                        counter: 0,
+                                        source_addr: packet.packet_association_table.source_addr,
+                                        destination_addr: packet.packet_association_table.destination_addr,
+                                        time: packet.time
+                                    });
+                            }
+                        }
+                    }
+                })
+            })
+        });
+        let keys = mpegts_rows.keys().collect::<Vec<_>>();
+        body.rows(25.0, mpegts_rows.len(), |row_ix, mut row| {
```

Comment:
What about [heterogeneous_rows](https://docs.rs/egui_extras/latest/egui_extras/struct.TableBody.html#method.heterogeneous_rows) to use height based on the amount of information in the fragment `PAT`, `PMT`, `PES`?